MCB-Nucleo/Core/Inc/MCB.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * MCB.h
+ *
+ *  Created on: Jun. 1, 2023
+ *      Author: kcgro
+ */
+
+#ifndef INC_MCB_H_
+#define INC_MCB_H_
+
+#include "can.h"
+
+#define CAN_DATA_LENGTH 8
+#define ADC_POLL_DELAY 10
+#define SEND_MOTOR_COMMAND_DELAY 10
```

Comment:
Same discussion on magic numbers, I would add a comment that just says milliseconds or use `SEND_MOTOR_COMMAND_DELAY_MS`. This may seem useless, but it takes practically no time and saves future you/future developers a short bit of time for code comprehension. Using more characters in a variable name for clarity is rarely a bad thing.

Reply:
Added comments for defines